translations/Readme.md

```diff
@@ -0,0 +1,30 @@
+# About Translation
+
+## How to use the translation utilities
+
+### On Windows
+
+#### 1. Adding .ts files
+
+1. Open `1-lupdate.cmd` in editer. (e.g, VSCode or Notepad, etc.)
+2. Append your new file name to the end of first line, in the format **locale_**_language-code_**.ts** (e.g, locale_de, local_zh-cn, etc.).
+3. Run the command file in cmd.
+4. You will find the new file in the same location as the `1-lupdate.cmd`.
+
+#### 2.Generating / Updating .qm files
+
+Just run the command file in cmd.
```

Comment:
plural, it is 2 files, one for generating/updating the ts files and another for generating the qm file

Reply:
Yes, it is. But ,in fact, each time when we restart the scripts￼ they updates existed files.

Reply:
I will specify them in details.

Reply:
￼ I found I copied the latter part of the sentence from 1.3. ￼